dotCMS/src/main/java/com/dotmarketing/portlets/workflows/util/WorkflowImportExportUtil.java

```diff
@@ -204,6 +209,23 @@ public void importWorkflowExport(final WorkflowSchemeImportExportObject importer
 		}
 	}
 
+	private Role getAnonRole () {
+		return Try.of(()->APILocator.getRoleAPI().loadCMSAnonymousRole()).getOrNull();
+	}
+
+	private WorkflowAction validateAction(final WorkflowAction action) {
```

Comment:
What other foreign keys are there? Should we also validate the next stepId?

Reply:
Yeah next steps should be ok, since they are not references, they are part of the import object.
The issue with next assign and permissions is that they are not auto contained in the import json, they are just references to existing objects on the db, so they can or not exists